### PR TITLE
Added new tables , query and joined tables

### PR DESCRIPTION
**Pull request by mohashyne**
# New Features Added
- [ ] Created a table called "owners" with the following columns;
    - id; with a primary key
   -  full_name; a string
   -  age;  owner's age

- [ ] Created a table called "species" with the following columns;
       - id; with a primary key
      -  name; a string
      
- [ ] Alter and  update the "animals" table, 
   - species_id  will be a key referencing the "species" table.
   - owner_id will be a foreign key referencing the "owners" table.

# Below are some query quests:

- [ ] Insert the following data into the owners table:
Sam Smith 34 years old.
Jennifer Orwell 19 years old.
Bob 45 years old.
Melody Pond 77 years old.
Dean Winchester 14 years old.
Jodie Whittaker 38 years old.

- [ ] Insert the following data into the species table:

Pokemon
Digimon

- [ ] Modify your inserted animals so it includes the species_id value:

If the name ends in "mon" it will be Digimon
All other animals are Pokemon

- [ ] Modify your inserted animals to include owner information (owner_id):
Sam Smith owns Agumon.
Jennifer Orwell owns Gabumon and Pikachu.
Bob owns Devimon and Plantmon.
Melody Pond owns Charmander, Squirtle, and Blossom.
Dean Winchester owns Angemon and Boarmon.

- [ ] Write queries (using JOIN) to answer the following questions:
What animals belong to Melody Pond?

- [ ] List of all animals that are Pokemon (their type is Pokemon).

List all owners and their animals, remember to include those that don't own any animal.

- [ ] How many animals are there per species?
- [ ] List all Digimon owned by Jennifer Orwell.
- [ ] List all animals owned by Dean Winchester that haven't tried to escape.
- [ ] Who owns the most animals?

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -35,4 +35,44 @@ VALUES ('Blossom', '1998-10-13', 3, true, 17);
 
 
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
-VALUES ('Ditto', '2022-05-14', 4, true, 22);
\ No newline at end of file
+VALUES ('Ditto', '2022-05-14', 4, true, 22);
```
Comment (Lucas-Masaba) [inline data.sql:38]: - [ ] Ido not see the following data that is supposed to be added to owners' table:
 - Sam Smith 34 years old.
 - Jennifer Orwell 19 years old.
 - Bob 45 years old.
 - Melody Pond 77 years old. 
 - Dean Winchester 14 years old.
 - Jodie Whittaker 38 years old.

**Review by Lucas-Masaba — CHANGES_REQUESTED** [review]
# Hello @mohashyne  👋 , 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## Highlights 🍾 
- All SQL is in its respective file :file_cabinet: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 💭 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (mohashyne) [reply data.sql:38]: Data update

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @mohashyne , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Well done adding the remaining of the data to the owner table. ✔
- The milestone requirements have been met with success. 🎉 🎉 🎉 ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**